### PR TITLE
Fix bugs thanks to Findbugs

### PR DESCRIPTION
**Pull request by geofjamg**

### CODE REVIEW

File: commons/src/main/java/eu/itesla_project/commons/config/MapModuleConfig.java [hunk 1]
```diff
@@ -135,6 +135,12 @@ public Integer getOptionalIntProperty(String name) {
     }
 
     @Override
+    public Optional<Integer> getOptionalIntProperty2(String name) {
```
Comment (mathbagu) [inline commons/src/main/java/eu/itesla_project/commons/config/MapModuleConfig.java]: Rename to getOptionalIntegerProperty?

File: commons/src/main/java/eu/itesla_project/commons/config/ModuleConfig.java [hunk 2]
```diff
@@ -37,8 +38,11 @@
 
     int getIntProperty(String name);
 
+    @Deprecated
     Integer getOptionalIntProperty(String name);
 
+    Optional<Integer> getOptionalIntProperty2(String name);
```
Comment (mathbagu) [inline commons/src/main/java/eu/itesla_project/commons/config/ModuleConfig.java]: Rename to getOptionalIntegerProperty

File: commons/src/main/java/eu/itesla_project/commons/config/MapModuleConfig.java (hunk 1 above)
Comment (geofjamg) [reply commons/src/main/java/eu/itesla_project/commons/config/MapModuleConfig.java]: Done

File: commons/src/main/java/eu/itesla_project/commons/config/ModuleConfig.java (hunk 2 above)
Comment (geofjamg) [reply commons/src/main/java/eu/itesla_project/commons/config/ModuleConfig.java]: Done

**Pull request merged**